paas/env_variables/staging.yml

```diff
@@ -46,3 +46,79 @@ direct_debit_frontend_sentry_dsn: ''
 direct_debit_frontend_session_encryption_key: asdjhbwefbo23r23rbfik2roiwhefwbqw
 direct_debit_frontend_url: directdebit-frontend.staging.gdspay.uk
 space: staging
+db_password: mysecretpassword
+db_ssl_option: ssl=false
+ledger_route: ledger-staging.apps.internal
+products_route: products-staging.apps.internal
+products_api_token: g0nBA5lNzfUalQjk4ZBDy70AikNshxz2G70dA6lg
```

Comment:
Fine to have stub values in for secrets at the moment. I guess the mid-term plan is to put these into concourse's SSM and pass them in that way? Longer term, they should probably be written to the credhub that doesn't exist yet.